### DISCUSSION TITLE
https://notes.junghanacs.com/notes/20240823t175024

### DISCUSSION THREAD

**Discussion by giscus** [junghanacs/notes.junghanacs.com#12] [Announcements]
# https://notes.junghanacs.com/notes/20240823t175024

모 가댓 행복을 풀다 - 구글X 공학자가 찾은 삶과 죽음 너머 진실 강주헌 (모 가댓 2017) Solve for Happy 책소개 구글 공동 창업자 세르게이 브린, 이어령, 이해인, 하지현, 김미경 강력 추천! 아마존 베스트셀러! 1,000만 명 행복 프로젝트 우리는 애초부터 행복하게 살도록 설계되어 있다 하지만 지금 행복한가? 세계적 공학자인 [구글X] ...

https://notes.junghanacs.com/notes/20240823t175024

**Comment by junghan0611** [comment]
https://gyazo.com/5bd4404cb0bb0f909f88933b45681003

**Reply by junghan0611** [reply]
이미지는 이런식으로 올려야 겠구나 무료서비스

**Comment by junghan0611** [comment]
![ima](https://gyazo.com/5bd4404cb0bb0f909f88933b45681003)
